CHANGELOG.md

```diff
@@ -19,6 +19,14 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Support for `DOCKER_HOST` parsing when using `docker-daemon://`
 - `DOCKER_USERNAME` and `DOCKER_PASSWORD` supported without `APPTAINER_` prefix.
 
+- Added support for signing and validating images using X509 certificates. For
+  the signing the flags --pkcs8Key and --x509cert are added. In the respective
+  order, the flags load the private key and the certificate of the authority
+  signing the image (signer). For the validation, --x509cert, --x509RootCA,
+  and --x509IntermediateCerts. In the respective order, the load the signer's
```

Comment:
Change "the load" to either "they load" or "the flags load"

Reply:
done

Reply:
should be ok now